### PR TITLE
Updated tests to not fail when the Plone Site root is dexterity.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
This is what is left over from https://github.com/plone/plone.restapi/pull/1200

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]
LGTM.

**Comment by mauritsvanrees** [comment]
Green. I think this can be merged now, independent of the other site root PRs, and without waiting for framework team approval.

**Pull request merged**
